csm_web/scheduler/views.py

```diff
@@ -170,13 +170,13 @@ def attendances(self, request, pk=None):
 
 
 class ProfileViewSet(*viewset_with('list')):
-    queryset = None
     serializer_class = None
 
+    def get_queryset(self):
+        pass
+
     def list(self, request):
-        student_profiles = StudentSerializer(request.user.student_set.filter(active=True), many=True).data
-        mentor_profiles = MentorSerializer(request.user.mentor_set.exclude(section=None), many=True).data
-        return Response({'mentor_profiles': mentor_profiles, 'student_profiles': student_profiles})
+        return Response(ProfileSerializer([*request.user.student_set.filter(active=True), *request.user.mentor_set.exclude(section=None)], many=True).data)
```

Comment:
I know we can't use the queryset `union` method because they're from two different tables, but this method of concatenating two different querysets feels weird. Do you know if the + operator or something similar can be used here instead of a spread?

Reply:
We could use `itertools.chain` to combine to make a single iterator over the two querysets which is passed to `ProfileSerializer`.

Reply:
I guess the current way is more readable, so we can just keep it as is.